CMS-Enterprise/easi-app
# [EASI-4554] GRB participation needed banner

# EASI-4554

[Figma](https://www.figma.com/design/G0UN35V7SY0g9nkTIESxsY/EASi-GRB-Experience?node-id=2243-37173&t=wPVCLV3RuT9N7w31-0)

## Description
- GRB participation needed banner
  - Only displays if user has been added as GRB reviewer to intake
- Table to display intakes where review has been requested
  - Default sort by GRB date:
    - No GRB date first
    - Past dates at end
    - Future dates sorted by closest date first  


## How to test this change
- Log in as `ABCD` with `EASI_USER` job code
- GRB participation banner should render and match Figma
- To test pagination and sorting, edit src/components/GrbParticipationNeeded/index.tsx to use mock data:
  - Change line 65 to `const systemIntakes = systemIntakesWithReviewRequested;`
  - Add `import { systemIntakesWithReviewRequested } from 'data/mock/systemIntake';` to imports

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
